server/src/main/java/org/apache/druid/segment/realtime/SegmentGenerationMetrics.java

```diff
@@ -55,6 +56,13 @@ public class SegmentGenerationMetrics
 
   private final AtomicLong maxSegmentHandoffTime = new AtomicLong(NO_EMIT_SEGMENT_HANDOFF_TIME);
 
+  // Message gap accounting
+  private final AtomicLong minMessageGap = new AtomicLong(Long.MAX_VALUE);
+  private final AtomicLong maxMessageGap = new AtomicLong(0);
```

Comment:
Did you find `new LongAccumulator(Math::max, Long.MIN_VALUE)` to be slower or no different from AtomicLong here?